### PR TITLE
allow the sync action to work with bootstrapper

### PR DESCRIPTION
**Pull request by joshzarrabi**
Signed-off-by: Daniel Thornton <dthornton@vmware.com>

### CODE REVIEW

File: actions/sync/entrypoint [hunk 1]
```diff
@@ -37,11 +38,12 @@ sync_dirs() {
   set -x
   cd "${config_repo}${config_path}"
     for dir in $(ls -Ap . | grep "/"); do
-      if [[ -f "${current_repo}/${dir}.syncignore" ]]; then
-        rsync -r "${dir}" "${current_repo}/${dir}" --delete \
-          --exclude=".syncignore" --exclude-from="${current_repo}/${dir}.syncignore"
+      local exclude_list=""
+      if [[ -f "${current_repo}${current_path}/${dir}.syncignore" ]]; then
+        rsync -r "${dir}" "${current_repo}${current_path}/${dir}" --delete \
+          --exclude=".syncignore" --exclude-from="${current_repo}${current_path}/${dir}.syncignore"
```
Comment (ryanmoran) [inline actions/sync/entrypoint:44]: You should always include a `/` between path segments. Extra `/` characters back-to-back do not hurt, but missing `/` characters means that your script breaks.

```suggestion
      if [[ -f "${current_repo}/${current_path}/${dir}.syncignore" ]]; then
        rsync -r "${dir}" "${current_repo}/${current_path}/${dir}" --delete \
          --exclude=".syncignore" --exclude-from="${current_repo}/${current_path}/${dir}.syncignore"
```

File: actions/sync/entrypoint [hunk 2]
```diff
@@ -37,11 +38,12 @@ sync_dirs() {
...
       else
-        rsync -r "${dir}" "${current_repo}/${dir}" --delete
+        rsync -r "${dir}" "${current_repo}${current_path}/${dir}" --delete
```
Comment (ryanmoran) [inline actions/sync/entrypoint:46]:
```suggestion
        rsync -r "${dir}" "${current_repo}/${current_path}/${dir}" --delete
```

Comment (joshzarrabi) [reply actions/sync/entrypoint:46]: I think we don't want it just incase its an empty string?

Comment (ryanmoran) [reply actions/sync/entrypoint:46]: That's totally fine. Bash will see a path like `/some-org/some-repo//some-dir` and convert that to `/some-org/some-repo/some-dir`.

**Pull request merged**